### PR TITLE
Pass 2 of addressing review comments in ITE-2

### PR DESCRIPTION
**Pull request by adityasaky**
I still need to structure the root-layouts-can-differ in, I'll push here shortly.

### CODE REVIEW

File: ITE/2/README.adoc [hunk 1]
```diff
@@ -169,21 +169,23 @@ are three important considerations regarding the targets metadata signed
...
-. Since each package is typically produced independently of other
-packages, this means that the complete set of in-toto _link_ metadata
-for a package SHOULD be different, and therefore isolated, from that for
-another package. (The in-toto _root layout_ metadata SHOULD be the same
-for every package.) This MAY be done in a number of different ways, but
-perhaps the simplest is to use directories with unique names to isolate
-different in-toto link metadata for different packages, where the name
-MAY be the SHA-256 hash of the "`developer`" step link metadata file for
-a particular package.
+. Since each package for a project is typically produced independently
```
Comment (adityasaky) [inline ITE/2/README.adoc:172]: I added "for a project" to help clarify that different packages are still from the same abstract supply chain.

File: ITE/2/README.adoc [hunk 2]
```diff
@@ -169,21 +169,23 @@ are three important considerations regarding the targets metadata signed
...
+of other packages, this means that the complete set of in-toto _link_
+metadata for a package SHOULD be different, and therefore isolated,
+from that for another package. This MAY be done in a number of different
+ways, but perhaps the simplest is to use directories with unique names to
+isolate different in-toto link metadata for different packages, where the
+name MAY be the SHA-256 hash of the "`developer`" step link metadata file
+for a particular package. The root layout MAY be identical for each
```
Comment (adityasaky) [inline ITE/2/README.adoc]: Does this bit make sense?

File: ITE/2/README.adoc [hunk 3]
```diff
@@ -169,21 +169,23 @@ are three important considerations regarding the targets metadata signed
...
+of these packages, but it can differ to allow for changes in the supply
```
Comment (trishankatdatadog) [inline ITE/2/README.adoc]: Could we please mention here that using consistent snapshots and custom targets metadata in TUF lets us trivially associate different packages (whether in the same project or not) with different root layout w/o breaking backwards compatibility?

**Review by trishankatdatadog — COMMENTED** [review]
This is excellent, thanks, Aditya! Would you please kindly address my comment?

File: ITE/2/README.adoc (hunk 3 above)
Comment (adityasaky) [reply ITE/2/README.adoc]: Does https://github.com/trishankatdatadog/ITE/pull/3/commits/df8c94889d1605bd49697d266db18a83a161abee look ok?

Comment (adityasaky) [reply ITE/2/README.adoc]: Actually not so much I think. Let me try again.

**Review by trishankatdatadog — APPROVED** [review]
💯, thanks, Aditya!
